datastore-v1-proto-client/src/test/resources/META-INF/native-image/com/google/datastore/v1/client/reflect-config.json

```diff
@@ -0,0 +1,83 @@
+[
```

Comment:
These configurations are needed for the `google-api-client` dependency which is not restricted to the `test` scope. How about we relocate this config file to `src/main/resources/...`?